CHANGELOG.rst

```diff
@@ -8,7 +8,7 @@ This document describes changes between each past release.
 
 ### Changed
 
-- Use a hashed ADMIN_PASSWORD instead of a clear text one (#236)
+- **BREAKING CHANGE** Use a hashed ADMIN_PASSWORD instead of a clear text one, see [doc](https://ihatemoney.readthedocs.io/en/latest/installation.html#configuration) for migration (#236)
```

Comment:
I think this is a good idea to state it's a breaking change. I would also write a "how to migrate" here, without having to follow a link.